.gitignore

```diff
@@ -9,3 +9,4 @@ lang/
 .pnp.*
 junit.xml
 .env.local
+/.idea/**
```

Comment:
Should we really add IDE-specific files to the Comet gitignore?
I have it in my globally configured gitignore for instance 🤷🏼 

@johnnyomair

Reply:
This is also removed in a follow up PR